tico/passes/decompose_fake_quantize_tensor_qparams.py

```diff
@@ -98,7 +102,7 @@ def get_constant_from_tensor(
         lifted_tensor_constants = ep.graph_signature.inputs_to_lifted_tensor_constants
         assert lifted_tensor.name in lifted_tensor_constants
         tensor_name = lifted_tensor_constants[lifted_tensor.name]
-        value = ep.constants[tensor_name].cpu().detach().numpy()
+        value = ep.constants[tensor_name].item()
```

Comment:
Another bug fix. Calling `numpy()` makes a Tensor to a numpy arrapy, which is not the case for single-element tensor.